client/components/mobile-phone-verification/mobile-phone-verification.js

```diff
@@ -156,13 +158,20 @@ export default class MobilePhoneVerification extends React.Component {
         phone_number,
       }),
     })
-      .then(() => {
+      .then((response) => {
         // flag SMS as sent to avoid resending it
         sessionStorage.setItem(self.phoneTokenSentKey, true);
         toast.info(t`TOKEN_SENT`);
+        if (response && response.data && response.data.timeout) {
+          this.setState({resendButtonDisabledTimeout: response.data.timeout});
```

Comment:
let's rename timeout as indicated in openwisp-radius, I think `cooldown` is easy to understand.